ww3tools/ProcSat_Altimeter.py

```diff
@@ -0,0 +1,440 @@
+   along the track.
+ This script uses confirguration inputs saved in ww3tools.yaml
+  and the reading module wread.py
+ A total of 15 satellite missions are listed below. The period of each
+  altimeter can be verified at:
+  https://www.sciencedirect.com/science/article/pii/S0273117721000594
+  https://ars.els-cdn.com/content/image/1-s2.0-S0273117721000594-gr1_lrg.jpg
+
+USAGE:
+ This program processes one satellite mission per run, entered as 
+  argument (only the ID, sys.argv), Select only one:
+  JASON3,JASON2,CRYOSAT2,JASON1,HY2,SARAL,SENTINEL3A,ENVISAT,ERS1,ERS2,GEOSAT,GFO,TOPEX,SENTINEL3B,CFOSAT
+ Altimeters must have been previously downloaded (see wfetchsatellite_AODN_Altimeter.sh)
+ User must inform if wants to target the model space (tspace=1) or the satellite
+  space (tspace=2) in ww3tools.yaml.
+ Quality control can be selected or excluded in ww3tools.yaml, and user
+  can modify satellite quality control parameters in ww3tools.yaml as well.
+ The path where AODN altimeter data is located must be informed in ww3tools.yaml
+ The output path where the results are saved must be informed in ww3tools.yaml
+ INPUT ARGUMENTS are (the first one is mandatory):
+ (1) satellite name
+ (2) initial date ('YYYYMMDDHH')
+ (3) final date ('YYYYMMDDHH')
+ (4) time-step for the time array of results
+ (5) ww3 grid output sample file or gridInfo.nc
+
+ The ww3tools.yaml file must be saved (or symbolic link) in the same dir.
+ Example (from linux terminal command line):
+   nohup python3 ProcSat_Altimeter.py JASON3 2018010100 209010100 3600 >> nohup_ProcSat_Altimeter_JASON3.out 2>&1 &
+
+OUTPUT:
+ netcdf file Altimeter_*.nc containing the processed altimeter data
+ hs: significant wave height, Ku or Ka altimeter band.
+ wnd: 10-meter wind speed.
+ 'cal' means calibrated by IMOS-AODN.
+
+DEPENDENCIES:
+ See setup.py and the imports below.
+ AODN altimeter data previously downloaded (see wfetchsatellite_AODN_Altimeter.sh)
+ Configure ww3tools.yaml as you prefer.
+
+AUTHOR and DATE:
+ 04/04/2022: Ricardo M. Campos, first version.
+ 07/18/2022: Ricardo M. Campos, SENTINEL-3B included. Longitude standard checked.
+ 05/21/2023: Ricardo M. Campos, CFOSAT included. HsC-band removed. New dlim default
+  obtained from https://doi.org/10.3390/rs15082203
+ 10/11/2023: Ricardo M. Campos, restructure to read config file with fixed inputs (.yaml),
+  modularization, altimeter data reading sent to wread.py. Renamed gridSatGlobal_Altimeter.py
+  to ProcSat_Altimeter.py
+ 10/13/2023: Ricardo M. Campos & Maryam Mohammadpour, include option to work on the
+  observation space. Averages along track.
+
+PERSON OF CONTACT:
+ Ricardo M Campos: ricardo.campos@noaa.gov
+ Maryam Mohammadpour: Ghazal.Mohammadpour@noaa.gov
+
+"""
+
+import numpy as np
+from matplotlib.mlab import *
+from pylab import *
+import pandas as pd
+import os
+import netCDF4 as nc
+import xarray as xr
+import pyresample
+from mpl_toolkits.basemap import shiftgrid
```

Comment:
I don't see where shiftgrid is used in this routine, can this import be removed?

Reply:
I agree. I've just corrected and pushed it

Reply:
Thanks!